src/components/FileContributors.tsx

```diff
@@ -119,29 +77,36 @@ const Contributor = ({ contributor }: { contributor: Author }) => {
 }
 
 export interface IProps extends FlexProps {
-  relativePath: string
   editPath?: string
   contributors: Array<Author>
-  lastContributor: any
   loading: Boolean
-  error: any
+  error?: boolean
   lastEdit: string
 }
 
 const FileContributors: React.FC<IProps> = ({
-  relativePath,
   editPath,
   contributors,
-  lastContributor,
   loading,
   error,
   lastEdit,
   ...props
 }) => {
   const [isModalOpen, setModalOpen] = useState(false)
-  const { language } = useI18next()
+  // TODO: Revert
+  // const { language } = useI18next()
+  const language = "en"
```

Comment:
I think we can `useRouter` now
```suggestion
  const { locale } = useRouter()
```

Reply:
Yep... #77 also addresses this; can patch this one here though